### PR TITLE
Drawing outside window#521

### PR DESCRIPTION
**Pull request by MrGumpIT**

### CODE REVIEW

**Comment by CodeDoctorDE** [comment]
i will test it tomorrow but here you should also remove the pubspec.lock change

File: app/lib/views/view.dart [hunk 1]
```diff
@@ -11,6 +11,7 @@ import 'package:flutter/gestures.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter/foundation.dart' as foundation;
```
Comment (CodeDoctorDE) [inline app/lib/views/view.dart:14]: Foundation is already imported in line 9, just use kIsWeb.

**Review by CodeDoctorDE — COMMENTED** [review]
I really don't see a difference between the current version and your changes.

**Comment by MrGumpIT** [comment]
You need to remove: https://github.com/LinwoodDev/Butterfly/issues/521#issuecomment-1920697889

**Comment by CodeDoctorDE** [comment]
Surprisingly it really works, thanks.
I really don't know why this bug exists.
I will merge this!

**Pull request merged**